samples/dashboards/component-options/sync-extremes/demo.js

```diff
@@ -0,0 +1,92 @@
+const CSVConnector = Dashboards.DataConnector.types.CSV;
+const csvData = document.getElementById('csv').innerText;
+
+const connector = new CSVConnector({
```

Comment:
The same applies to each demo you did ;)

Reply:
Done